prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/SC/sc-phd.yml

```diff
@@ -0,0 +1,95 @@
+---
+- name: "sc-phd"
+  description: "South Carolina Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "SC"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "sc-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform"
+        useTestProcessingMode: false
```

Comment:
Update schemaName to use classpath: . We also shouldn't need to have these file in multiple places. This file should be sufficient, the others can be removed.